main.js

```diff
@@ -0,0 +1,208 @@
+    image: '<img src="assets/Snapshoot Portfolio.jpg" class="img-pop-up" alt="portfolio-image">',
+    techs: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    live: 'http://bh-observer.me/projects/patatapcloneproject/index.html',
+    source: 'https://github.com/bhobserver/bhobserver.github.io/tree/main/projects/patatapcloneproject',
+  },
+  {
+    name: 'Multi-Post Stories <br> Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.<br><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+    image: '<img src="assets/Snapshoot Portfolio.jpg" class="img-pop-up" alt="portfolio-image">',
+    techs: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    live: 'http://bh-observer.me/projects/message/index.html',
+    source: 'https://github.com/bhobserver/bhobserver.github.io/tree/main/projects/message',
+  },
+  {
+    name: 'Multi-Post Stories <br> Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.<br><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+    image: '<img src="assets/Snapshoot Portfolio.jpg" class="img-pop-up" alt="portfolio-image">',
+    techs: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    live: 'http://bh-observer.me/projects/mazegame/index.html',
+    source: 'https://github.com/bhobserver/bhobserver.github.io/tree/main/projects/mazegame',
+  },
+  {
+    name: 'Multi-Post Stories <br> Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.<br><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+    image: '<img src="assets/Snapshoot Portfolio.jpg" class="img-pop-up" alt="portfolio-image">',
+    techs: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    live: 'http://bh-observer.me/projects/colorgame/index.html',
+    source: 'https://github.com/bhobserver/bhobserver.github.io/tree/main/projects/colorgame',
+  },
+];
+
```

Comment:
I'd kindly suggest that you add some margin-bottom to the desktop view of your popup window to enale it have a better look and so it is close to the design given 😄 

Your design :
![image](https://github.com/BHObserver/my-portfolio/assets/20537712/592958d3-7736-4070-b28e-b54023e5d69a)

Figma design :
![Uploading image.png…]()